package.json

```diff
@@ -9,7 +9,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@11ty/eleventy": "3.0.0-alpha.14",
+    "@11ty/eleventy": "^3.0.0-alpha.14",
```

Comment:
Please don't change this line so we always get the same version of this package every time.